.github/workflows/rit.yml

```
@@ -0,0 +1,51 @@
+name: Rootstock Integration Tests
+
+on:
+  push:
+    branches: ["master", "*-rc"]
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches: [ "master", "*-rc" ]
+  workflow_dispatch:
+    inputs:
+      rit-branch:
+        description: 'Branch for Rootstock Integration Tests'
+        required: false
+        default: 'main'
+      rskj-branch:
+        description: 'Branch for RSKJ repo'
+        required: false
+        default: 'master'
+
+jobs:
+  rootstock-integration-tests:
+    name: Rootstock Integration Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Set Branch Variables
+        id: set-branch-variables
+        run: |
+          # Default values
+          POWPEG_BRANCH="master"
+          RIT_BRANCH="${{ github.event.inputs.rit-branch || 'main' }}"
+          RSKJ_BRANCH="${{ github.event.inputs.rskj-branch || 'master' }}"
```

Is there a way to somehow re-use the default branch values that are defined above?

---

Hello, yes, it seems possible. Do you think it's better to re-use it? Because anyway, to change these default values we would have to change the wofklow yaml anyway.

---

Hello, yes, it seems possible. Do you think it's better to re-use it? Nevertheless, to change these default values we would have to change the worfklow yaml anyway.


We would have to do something like this:


```suggestion
          RIT_BRANCH="${{ github.event.inputs.rit-branch || github.event.inputs['rit-branch'].default }}"
          RSKJ_BRANCH="${{ github.event.inputs.rskj-branch || github.event.inputs['rskj-branch'].default }}"
```

---

Ops, the suggestion failed. Probably because the inputs are define only if the pipeline is triggered via workflow_dispatch. This logic wouldn't work for the commits in PR.

Let's let as it was? Otherwise we will need an even bigger if like:

`RSKJ_BRANCH="${{ github.event.inputs.rskj-branch || github.event.inputs['rskj-branch'].default || 'main' }}"`

---

No big deal, I wouldn't expect the default branches to change anytime soon. Just the good practice of avoid repetition (DRY). But in this case I'd go with the simplest solution